Formula/kubeless.rb

```diff
@@ -0,0 +1,35 @@
+class Kubeless < Formula
+  desc "Kubernetes Native Serverless Framework"
+  homepage "https://github.com/kubeless/kubeless"
+  url "https://github.com/kubeless/kubeless.git",
+    :tag => "v0.2.3",
+    :revision => "bfefb404e22a209659ef06b98a51051d72dd5c21"
+  depends_on "go" => :build
+  depends_on "kubernetes-cli" => :recommended
+
+  def install
+    commit = `git rev-parse --short HEAD`.chomp
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/kubeless/kubeless").install Dir["*"]
+    cd "src/github.com/kubeless/kubeless" do
+      system "go", "build", "-o", bin/"kubeless", "-ldflags",
+      "-w -X github.com/kubeless/kubeless/version.VERSION=v#{version} " \
```

Comment:
Use [this sort of format for ldflags](https://github.com/Homebrew/homebrew-core/blob/6d2f9f6fd82923814ac99f56aed674a38dfa4ff7/Formula/ssh-vault.rb#L25-L26). Using `\` in formulae to split lines is just... grim.